src/Context/product-context.js

```diff
@@ -0,0 +1,637 @@
+import { useState, useReducer, useContext, createContext} from 'react'
+import HarryPotterDeathlyHollows from "../Assets/Images/Book_Covers/harry-potter-and-the-deathly-hallows.jpg"
+import Attack_On_Titan from "../Assets/Images/Book_Covers/Attack_On_Titan.jpg"
+import The_Fault_In_Our_Stars from "../Assets/Images/Book_Covers/The_Fault_in_Our_Stars.jpg"
+import Naruto_Volume1 from "../Assets/Images/Book_Covers/Naruto Volume-1.jpg"
+import Naruto_Volume72 from "../Assets/Images/Book_Covers/Naruto Volume-72.jpg"
+import DanBrown_DaVinciCode from "../Assets/Images/Book_Covers/Dan Brown- Da Vinci Code.jpg"
+import DanBrown_Angels_and_Demons from "../Assets/Images/Book_Covers/Dan Brown-Angels and Demons.jpg"
+import MansSearchForMeaning from "../Assets/Images/Book_Covers/Mans_Search_For_Meaning.jpg"
+import The_Secret from "../Assets/Images/Book_Covers/The_Secret.jpg"
+import Shades_Of_Grey from "../Assets/Images/Book_Covers/50_Shades_Of_Grey.jpg"
+import I_too_had_a_love_story from "../Assets/Images/Book_Covers/I_too_had_a_love_story.jpg"
+import The_Mystery_Of_The_Blue_Train from "../Assets/Images/Book_Covers/The_Mystery_Of_The_Blue_Train.jpg"
+import Death_On_The_Nile from "../Assets/Images/Book_Covers/Death_On_The_Nile.jpg"
+import Eloquent_JavaScript from "../Assets/Images/Book_Covers/Eloquent_JavaScript.jpg"
+import You_Dont_Know_JS from "../Assets/Images/Book_Covers/You_Dont_Know_JS.jpg"
+
+
```

Comment:
Suggestion: You can use a file where you can import all the images together, by the way this also looks fine

Reply:
Hmmm, for components I will do this but for Images I think it will only add extra file in project.